products/amazon-aurora-postgresql.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Amazon Aurora for PostgreSQL
+category: service
+tags: amazon managed-postgresql
+iconSlug: amazonaws
+permalink: /amazon-aurora-postgresql
+releasePolicyLink:
+  https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.VersionPolicy.html#Aurora.VersionPolicy.MajorVersionLifetime
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "15"
```

Comment:
We must decide to track either major or minor versions, as both have EOL dates. Note: major version also have an extended support date that cannot be documented (I think) if we use minor.

If major versions are tracked the following calendar must be used: https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html#Release.Calendar.

If minor versions are tracked the following calendar must be used: https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html#PostgreSQL.Concepts.VersionMgmt.Supported.

It's easier to follow major versions, but that may be not what you want as a user. WDYT @glenn-sontheimerk12 ?

Reply:
Think we should use https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.VersionPolicy.html. Above links are for Amazon RDS MySQL / PostgreSQL.

Reply:
Thanks @marcwrobel  for the feedback and work on this.

I think we need to track the minor versions for the following reason:

1. When a major version is identified as EOL AWS specifies an LTS minor version and enrolls the database into RDS Extended support for 3 years.  
2. RDS Extended support:   provides security patches which do not change the minor version.  
3. The main issue is that when minor versions are EOL'd AWS will force upgrade our databases and we need to control when those minor version upgrades happen.

If we track minor versions do we need to include every minor version in our configuration or can we just include the ones that go EOL? i.e. skip minor versions in the config?

I think your point of it being easier to track major versions is because tracking minor versions requires more frequent updates?  Is that correct?

Reply:
> If we track minor versions do we need to include every minor version in our configuration or can we just include the ones that go EOL? i.e. skip minor versions in the config?

We could skip some versions if needed.

> I think your point of it being easier to track major versions is because tracking minor versions requires more frequent updates? Is that correct?

Yes, that's correct. And in both cases it would be great if it can be automated.

So far we have the following sources:

- https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.VersionPolicy.html : document major releases with EOL for both MySQL / PostgreSQL, but minor releases with EOL only for MySQL.
- https://docs.aws.amazon.com/AmazonRDS/latest/AuroraPostgreSQLReleaseNotes/AuroraPostgreSQL.Updates.html : document the versions.

There is an issue for documenting minor version though : I did not find the page that document minor versions EOL dates.